src/Library/demos/Spinner/main.vala

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env -S vala workbench.vala --pkg gtk4 --pkg libadwaita-1
+
+public void main () {
+  Gtk.init ();
+
+  var button = workbench.builder.get_object ("button") as Gtk.Button;
```

Comment:
Please use here the (Gtk.Button) casting syntax.

Reply:
Changed.

I also checked other examples in the Library, and about half of them are using checked casts for the result of `builder.get_object()` whereas the other half is using unchecked casts. I guess that should be unified to not cause unnecessary confusion to beginners.

Reply:
hm. seems like I didnt review them (properly). That should be done in a separate PR though.